crates/aptos-telemetry-service/src/log_ingest.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    auth::with_auth,
+    clients::humio::{CHAIN_ID_TAG_NAME, EPOCH_FIELD_NAME, PEER_ID_FIELD_NAME, PEER_ROLE_TAG_NAME},
+    constants::MAX_CONTENT_LENGTH,
+    context::Context,
+    error::ServiceError,
+    types::{auth::Claims, humio::UnstructuredLog},
+};
+use aptos_config::config::PeerRole;
+use aptos_logger::{debug, error};
+use flate2::bufread::GzDecoder;
+use reqwest::{header::CONTENT_ENCODING, StatusCode};
+use std::collections::HashMap;
+use warp::{filters::BoxedFilter, reject, reply, Buf, Filter, Rejection, Reply};
+
+pub fn log_ingest(context: Context) -> BoxedFilter<(impl Reply,)> {
+    warp::path!("log_ingest")
+        .and(warp::post())
+        .and(context.clone().filter())
+        .and(with_auth(
+            context,
+            vec![PeerRole::Validator, PeerRole::Unknown],
```

Comment:
Should only Validator and ValidatorFullNode be accepted?

Reply:
Unknown is still authenticated but their identity is not verified from on chain config. We use this in leiu of VFN role because we cant distinguish VFNs yet. That PR is on its way.